.github/renovate.json

```diff
@@ -0,0 +1,23 @@
+{
+  "extends": [
+    "config:best-practices",
+    ":gitSignOff",
+    ":rebaseStalePrs",
+    "group:allNonMajor",
```

Comment:
Do you really want to have a single PR all non majors? 

https://docs.renovatebot.com/noise-reduction/#lock-file-considerations

Reply:
what would you use instead?

Reply:
In my experience "super" PRs are hardly ever merged because something is always breaking. Smaller PRs allow to better find the source of regressions and are faster to review as you have single changelog on your PR. 

People adopt auto merging usually pretty fast, at that at least for specific dependencies like dev ones. 
https://docs.renovatebot.com/noise-reduction/#automerging

In the end, my comment was meant as a check if that is the intended behavior.

Reply:
@vinzscam does your ❤️ mean "yes, remove `group:allNonMajor`" or does it mean "yes, we want a single PR for all non majors?" 

Easy to remove that line, just need to know what y'all prefer to see. ;)

Reply:
Sorry, it was more of a 'thank you for the recommendation.' I leave the decision to you all as I don't have enough experience on this matter. 😅

Reply:
ok, no worries.

Then I just need a +1 / lgtm on this PR and I can merge it. 

@secustor @vinzscam @freben @tudi2d @awanlin any final thoughts before I mash the `squash and merge` button?

Reply:
:shipit: